wgpu-hal/src/dx12/mod.rs

```diff
@@ -211,6 +212,7 @@ impl DeviceShared {
         self.cmd_signatures.destroy();
         self.heap_views.raw.destroy();
         self.heap_samplers.raw.destroy();
+        // self.rtv_pool.into_inner().destroy();
```

Comment:
This probably should come out (but see the overall review comments, maybe it'll go away entirely).